packages/compass-aggregations/src/modules/aggregation.ts

```diff
@@ -155,6 +159,23 @@ export const fetchNextPage = (): ThunkAction<
   };
 };
 
+export const retryAggregation = (): ThunkAction<
+  void,
+  RootState,
+  void,
+  Actions
+> => {
+  return (dispatch, getState) => {
+    const {
+      aggregation: { errorPage }
+    } = getState();
+    if (!errorPage) {
```

Comment:
I think this would make the aggregation not re-run if the `errorPage` is `0`? Should we check for null/undefined instead?
```suggestion
    if (!errorPage) {
```

Reply:
nice catch! but after @gribnoysup's suggestion, I removed `errorPage` from the state and am reusing `page` prop.